cas-server-support-jpa-service-registry/src/main/resources/META-INF/spring/jpa-svc-reg-context.xml

```diff
@@ -35,6 +35,9 @@
        http://www.springframework.org/schema/security http://www.springframework.org/schema/security/spring-security.xsd
        http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd">
 
+    <context:component-scan base-package="org.jasig.cas" />
```

Comment:
Valid point. I think most modules have this definition. I will keep this here for now, but in a separate issue, will go through to remove duplicate scans.